modules/extensions/src/mask/shader-module.ts

```diff
@@ -1,5 +1,5 @@
 import type {ShaderModule} from '@luma.gl/shadertools';
-import {project, UniformTypes} from '@deck.gl/core';
+import {project} from '@deck.gl/core';
```

Comment:
Just a personal preference nit: This `<module>/shader-module.ts` naming scheme leaves us with a lot of files with the same name. I find that confusing when searching. Would recommend calling this `mask-shader-module.ts` or something more unique.

Reply:
No strong opinions here, but for me the fuzzy searching works for the whole path

<img width="384" alt="Screenshot 2024-07-31 at 15 11 24" src="https://github.com/user-attachments/assets/bb65bc6d-0ecf-4b27-863b-1f26619be762">

Reply:
First: I am not worried about this, leaving it is fine. 

That said (bikeshed mode on):
- I looked and was happy to see that the other files in the directory have the name of the extension (`mask-`) in their name. So why not this file too? 
- Seems like two different naming philosophies are in play at the same time?  
- At least the naming scheme seems consistent the various new extensions...

Reply:
While we're bikeshedding, for whatever reason the `shader-module` files have an anonymous default export whereas the others do not:

```ts
class MaskPass extends LayersPass;
class MaskExtension extends LayerExtension;
class MaskEffect implements Effect;
```
There is a pattern here, where the full superclass is not used for brevity (e.g. we don't have `MaskLayersPass`). So I would argue if we want to rename the files it would be best to go with `MaskModule` and then `src/mask/mask-module.ts`

Reply:
> have an anonymous default export whereas the others do not:

Yes, I have virtually removed all default exports from the internals of the other visgl libraries, as default export are less safe, less compatible than named exports. It would be great to see deck go in the same direction.

> So I would argue if we want to rename the files it would be best to go with MaskModule and then src/mask/mask-module.ts

So we do need a name. 
- In luma.gl the shader modules are named with lower case and no suffix: `picking`, `project32` etc. 
- That naming convention is rather presumptive ("we think shader modules are so important that their names don't need any qualifications"). 
- Calling it `mask` would follow luma conventions, but would be a bit "stark" here?
- Calling it `maskModule` would be somewhere in-between, perhaps the most reasonable compromise?
- Calling it `MaskModule` additionally has the problem that I usually use PascalCase symbols for types, not objects.